config/locales/models/user/en.yml

```diff
@@ -0,0 +1,73 @@
+# Structure needed for Rails i18n active record messages management
+# Example file
+en:
```

Comment:
I would leave a few as example. You could add a comment about that above . Remember this is an open source library, It should be generic and well documented